charts/spire/charts/spire-server/templates/configmap.yaml

```diff
@@ -1,4 +1,4 @@
-{{ $namespace := .Release.Namespace }}
+{{ $server := . }}
```

Comment:
```suggestion
{{ $root := . }}
```

Is a common convention, as you refer to the root of the values. $server is misleading, as `.` refers to root of values.